### DISCUSSION TITLE
update files from Foxtrot to Golf

### DISCUSSION THREAD

**Discussion by nx2i** [kg4vdk/arcos-linux-modules#40] [It's Not In The Manual...Promise!] [answered]
Mike and others,
I am likely missing something, but I'll ask anyway. Is there a script or other capability to transfer saved QRV files specific to my callsign from a "Foxtrot" to a "golf" version of arcOS? I have already "manually" copied my YAAC map files, but right now I am faced with re-entering a bunch of config data for FLRIG, etc.
73, John, NX2I

**Comment by kg4vdk** [comment]
Hi John,

You didn't miss anything :-) There is no script to handle this...yet, maybe.

Here's the structure to help you copy your personal "default" configs you already have into their new correct location(s). You'll want to create `/ARCOS-DATA/QRV/CALLSIGN/SAVED/PROFILES/DEFAULT` + each of the applications you have configs for, as below.

```
/ARCOS-DATA/QRV/KG4VDK/SAVED/PROFILES/DEFAULT/FL-SUITE/fl-suite_DEFAULT.tgz
/ARCOS-DATA/QRV/KG4VDK/SAVED/PROFILES/DEFAULT/JS8CALL/JS8Call_DEFAULT.ini
/ARCOS-DATA/QRV/KG4VDK/SAVED/PROFILES/DEFAULT/PAT/config_DEFAULT.json
/ARCOS-DATA/QRV/KG4VDK/SAVED/PROFILES/DEFAULT/WSJTX/WSJT-X_DEFAULT.ini
/ARCOS-DATA/QRV/KG4VDK/SAVED/PROFILES/DEFAULT/YAAC/yaac-profile_DEFAULT
```

So, for example, if you have an `fl-suite.tgz`, you'd rename it to `fl-suite_DEFAULT.tgz`, and place it in `...SAVED/PROFILES/DEFAULT/FL-SUITE/`.

Hopefully, that gets you where you want to be!

73 de KG4VDK,
Mike

**Comment by nx2i** [comment]
Mike and others,
I have another issue with the upgrade. The "wifi issue" that you fixed for "Foxtrot" (see [https://github.com/kg4vdk/arcos-linux-modules/discussions/11#discussioncomment-10349012], is not working for me in "Golf". I extracted the tar file BROADCOM-B43 in the same manner as in Foxtrot and setup identically, It fails the load the module in "Golf". I cannot see a logfile to help figure this out.
73, John, NX2I

**Reply by kg4vdk** [reply, marked as answer]
I think this is due to a singular variable in the `00_BROADCOM-B43.sh` script file on line 17.

```
# PATHS
ARCOS_DATA=/media/$USER/ARCOS-DATA
```

The mount point for `ARCOS-DATA` changed in "Golf" to shorten the full path. So, in the script file, you will need to change the `ARCOS_DATA` variable from:

`ARCOS_DATA=/media/$USER/ARCOS-DATA`
to
`ARCOS_DATA=/ARCOS-DATA`

**Reply by nx2i** [reply]
Mike, thank you for the quick response. It works well now.
I am a big fan of linux, so I really appreciate your efforts! 
The D630 machine is a dual boot win7/mint21. I would love to abandon the win 7, but the locals here are married
to varafm. Sooner or later......
73,
John, NX2I
